source/code-generation/query-converter/manage-queries.txt

```diff
@@ -0,0 +1,41 @@
+.. _rm-manage-queries:
+
+==============
+Manage Queries
+==============
```

Comment:
(info typing)
The title (starting with a verb) suggests task page. Info inside is reference. Do we need both task and reference? In this case we might want to retitle. Query Management. Or Query Management Options. Or something.